README.md

```diff
@@ -8,7 +8,7 @@ Person A should modify content here.
 
 ## Person B:
 
-Person B should modify content here.
+Am I the person I thought I'd be?
```

Comment:
Good question! Love your imagination on this line, keep up the good work and never change!!